<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linear Equations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="inputLine">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Example: 4 -2 0 3.5  |  last line is for last column of your matrix!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="removeLine">
       <property name="text">
        <string>Remove Selected Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addLine">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <item>
      <property name="text">
       <string>1.5 2 1.5 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.0 2.0 4.0 1.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1. 6.0 0 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.0 1.0 4.0 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 6 7 8</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calculateButton">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="outputLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
